autogluon/requirements.txt

```diff
@@ -1,2 +1,7 @@
- pandas==1.3.3
- autogluon==0.3.1
+--find-links https://download.pytorch.org/whl/torch_stable.html
+torch==1.9.1+cpu
+torchvision==0.10.1+cpu
+pandas==1.3.3
+numpy==1.19.5
+pillow>=6.2.0
+autogluon.tabular[all]==0.3.1
```

Comment:
unless there are plans to use vision and text multi-modal setup (needs GPUs), this should be enough for tabular problems testing.

Reply:
Also add MXNet and sklearnex:

```
autogluon.tabular[all,skex]==0.3.1
mxnet<2.0.0
```